src/main/python/DataAnalysis/DiffusionParameterData.py

```diff
@@ -79,7 +102,47 @@ def get_combined_param_region_summary(self, param_name, patient_to_regions):
         if param_name in scale_params:
             for i in range(1, len(summary)):
                 summary[i] = summary[i] * 1000
-        return summary
+
+        # if the parameter is HA_range, return also the min and max
+        if param_name == 'HA_range':
+            min_HA_range = param_values_as_series.min()
+            max_HA_range = param_values_as_series.max()
+            return summary, [min_HA_range, max_HA_range]
+
+        elif param_name == "HA_bins":
+            LHC, CHC, RHC = 0, 0, 0
+            if param_values_as_series.size != 0:
+                LHC = param_values_as_series[(-90 <= param_values_as_series) & (
+                        param_values_as_series < -30)].count() / param_values_as_series.size
+                CHC = param_values_as_series[(-30 <= param_values_as_series) & (
+                        param_values_as_series <= 30)].count() / param_values_as_series.size
+                RHC = param_values_as_series[(30 < param_values_as_series) & (
+                        param_values_as_series <= 90)].count() / param_values_as_series.size
+            return summary, [LHC, CHC, RHC]
+
+        elif param_name == "IA_bins":
```

Comment:
I cannot spot a difference between these elifs if this is the case can you create a method to avoid duplication?

Reply:
Or just make one elif statement with ors